com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/PollServiceDeleteOperationsExecution.java

```diff
@@ -43,13 +44,15 @@ private ServiceOperation getLastDeleteServiceOperation(ExecutionWrapper executio
             return null;
         }
         boolean isServiceDeleted = isServiceDeleted(execution, service.getMeta()
-            .getGuid());
+                                                                      .getGuid());
```

Comment:
Why this is formatted?

Reply:
There was an agreement to change the formatting style.